core/print_api.php

```diff
@@ -1654,7 +1674,7 @@ function get_email_link( $p_email, $p_text ) {
  * @param string $p_bug_id The bug identifier.
  * @return void
  */
-function print_email_link_with_subject( $p_email, $p_text, $p_tooltip, $p_bug_id ) {
+function print_email_link_with_subject( $p_email, $p_text, $p_tooltip, $p_bug_id, $p_icon_only = false ) {
```

Comment:
Parameter needs to be documented (PHPDoc).
I also suggest to rename it to `$p_show_tooltip = true` to better reflect its purpose.